modules/ztp-sno-siteconfig-config-reference.adoc

```diff
@@ -51,6 +51,9 @@ For standard deployments, define three hosts with `role: master` and two or more
 |`spec.clusters.nodes.nodeLabels`
 |Specify custom roles for your nodes in your managed clusters. These are additional roles are not used by any {product-title} components, only by the user. When you add a custom role, it can be associated with a custom machine config pool that references a specific configuration for that role. Adding custom labels or roles during installation makes the deployment process more effective and prevents the need for additional reboots after the installation is complete.
 
+|`spec.clusters.nodes.automatedCleaningMode`
+|Optional. If the value is set to `metadata`, the partitioning table of the disk is removed, but the disk is not fully wiped. By default, the value of the `automatedCleaningMode` field is `disabled`. To enable removing the partitioning table, uncomment this line and set the value to `metadata`.
```

Comment:
```suggestion
|`spec.clusters.nodes.automatedCleaningMode`
|Optional.  Uncomment this line and set the value of the `automatedCleaningMode` field to `metadata` to enable the removal of the disk's partitioning table. However, the removal does not completely wipe the disk. The default value is `disabled`.
```
Suggest starting with action words in active voice and present tense. Minimalism principles 1 & 4 [Cutomer focus, action orientation, elimination of fluff] help here. It would be nice to ensure parallelism and consistency in writing style. This helps in a unified user experience. TIP: Please read through the definitions and descriptions in the table and observe the writing style.

**Refer to:** https://redhat-documentation.github.io/supplementary-style-guide/#minimalism and https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc

Reply:
Thank you! Just a little tweak for the `However, the removal does not completely wipe the disk.` part. WDYT?
> Optional. Uncomment and set the value to `metadata` to enable the removal of the disk's partitioning table only, without fully wiping the disk. The default value is `disabled`.

Reply:
@amolnar-rh Love the change! Please go ahead. Thank you!